src/components/RegistrationPage/RegistrationForm.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect } from 'react'
+import { useSelector, useDispatch } from "react-redux";
+import { actionsCategories, selectorsCategories } from 'store/sliceCategories'
+import { AppDispatch } from 'store/store';
+import { CheckOutlined, CloseOutlined } from '@ant-design/icons';
+import { useNavigate } from "react-router-dom";
+import { Alert } from 'antd';
+import {
+  Form,
+  Input,
+  Button,
+  Radio,
+  DatePicker,
+  Switch,
+  Checkbox
+} from 'antd';
+import { Col, Row, Divider } from 'antd';
+import type { CheckboxChangeEvent } from 'antd/es/checkbox';
+import css from './registrationForm.module.css'
+export const RegistrationForm: React.FC<{}> = () => {
+  const navigate = useNavigate();
+  const categoriesItems = useSelector(selectorsCategories.getCategories);
+  const dispatch = useDispatch<AppDispatch>();
+  useEffect(() => {
+    dispatch(actionsCategories.actions.fetchCategories())
```

Comment:
```suggestion
    dispatch(actionsCategories.fetchCategories())
```
как-то странно выглядит actions.actions